packages/store/src/types.js

```diff
@@ -750,11 +750,19 @@
  */
 
 /**
- * @typedef {any} MethodGuardMaker
- * a parameter list like foo(a, b, c = d, …e) => f should be guarded by
- * something like
- * foo: M.call(AShape, BShape).optional(CShape).rest(EShape).returns(FShape)
- * optional is for optional (=) params. rest is for … (varargs) params
+ * @typedef {object} MethodGuardMaker
+ *
+ * A parameter list and return value like:
+ * ```js
+ *   foo(a, b, c = d, ...e) => f
+ * ```
+ * should be guarded by something like:
+ * ```js
+ *   M.call(AShape, BShape).optional(CShape).rest(EShape).returns(FShape)
+ * ```
```

Comment:
```suggestion
 * A method name and parameter/return signature like:
 * ```js
 * foo(a, b, c = d, ...e) => f
 * ```
 * should be guarded by something like:
 * ```js
 * {
 *   ...otherMethodGuards,
 *   foo: M.call(AShape, BShape).optional(CShape).rest(EShape).returns(FShape),
 * }
 * ```
```